gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #217: Set debug to false

### Description

* Set debug to false
* Used whitenoise library to handle static files

### Testing Directions
1. Run `python manage.py collectstatic`
1. I deployed this branch to our develop Heroku instance. So go to https://showup-nyc-develop.herokuapp.com/ and see that everything loads correctly. Our images are shown so that proves that the static files are being handled properly.
1. Go to an incorrect URL on our site such as https://showup-nyc-develop.herokuapp.com/asdfldfdh and see that you get a generic error page instead of the Django debug page, as desired
![image](https://user-images.githubusercontent.com/13837978/70296412-8c179b00-17b8-11ea-9323-c0b13eff8376.png)


### Miscellaneous Comments

* I followed some of the instructions here: https://devcenter.heroku.com/articles/django-assets

### Checklist
